bitmap.h

```diff
@@ -9,6 +9,7 @@
 #define BIT_CLR(n, b) ((n) &= ~(1 << (b)) )
 #define BIT_INV(n, b) ((n) ^=  (1 << (b)) )
 #define BIT_GET(n, b) (((n) & (1<<(b))) != 0)
+#define BITS_GET(n, l, b) ( ((n) & (( (1<<(l)) - 1) << (b) )) >> b)
```

Comment:
Oops... actually those are defined in `mips-mti-elf/include/mips/mips32.h` from toolchain directory, please use them.